openmetadata-ui/src/main/resources/ui/src/utils/EntityUtils.tsx

```diff
@@ -1333,6 +1337,10 @@ export const getEntityLinkFromType = (
       );
     case EntityType.TEST_CASE:
       return getIncidentManagerDetailPagePath(fullyQualifiedName);
+    case EntityType.DOMAIN:
+      return getDomainDetailsPath(fullyQualifiedName);
```

Comment:
Methods says getEntityLink but we are returning details path
We can use method from EntityClass for getting details path